# Modifications to SparsePostQuantumRatchet source code

This file contains the diff between the original upstream source
and the modified version used in this verification project.

Upstream Repository: https://github.com/signalapp/SparsePostQuantumRatchet
Upstream Commit: f2589fef855c10f39d72634dab3d14654dd410bf

---

diff -Naur --no-dereference a/src/.DS_Store b/src/.DS_Store
--- a/src/.DS_Store
+++ b/src/.DS_Store
@@ -0,0 +1 @@
